lib/standards.h

```diff
@@ -37,7 +37,7 @@ struct Standards {
     enum Language { None, C, CPP };
 
     /** C code standard */
-    enum cstd_t { C89, C99, C11, CLatest = C11 } c = CLatest;
+    enum cstd_t { C89, C99, C11, /*, C17*/ C23, CLatest = C23 } c = CLatest;
```

Comment:
Please also enable the GUI part and feel free too fully add C17. I somehow forgot about that - it is supported upstream in simplecpp.

Reply:
Turns out there is no disabled GUI code. So I will take care of both things in a separate PR instead. Sorry about the noise.

Reply:
What do you mean by GUI part? I missed the `getOnly()` function, will add.
C17 doesn't seem to change much, so we'll probably not need an enum value for it: https://en.cppreference.com/w/c/17

Reply:
Just disregard the comment about the GUI.

The reason it did not add any new language features is most likely the reason I omitted it. But we still need to add it since it changes the standard define string.